### ISSUE TITLE
Require dates on sessions?

### ISSUE THREAD

**Issue by estaub** [opencivicdata/pupa#314]
I believe it would be useful to enforce that all sessions from 2017 have a start date, and that "old" sessions from 2017 and beyond must also have an end date.  

I've struggled to come up with a good definition for an "old" session.  The best limit I can come up with is to say that there can be no more than two sessions after 2017 without end dates.  This is wrong if there are ever any cases where more than one special session can be in session.

**Comment by showerst** [comment]
What do you propose in case of prefiling, where a session start date isn't set yet? Could always set the current date and change it later, but that feels a bit hacky.

**Comment by estaub** [comment]
@showerst Yes, I think setting an approximate stopgap start date is fine.  I've never seen this case - is it usually for special sessions?  

Even when "final", dates seem to be a bit iffy, so a little more doesn't concern me.  In doing all the additions a couple of months ago, there were several cases where I had conflicting inputs, especially for end dates.  In general, I opted for the earlier start-date or later end-date.

**Comment by showerst** [comment]
@estaub makes sense. I'll add them going forward. PS thanks for doing this!
